mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/tiltaksgjennomforinger/TiltaksgjennomforingValidatorTest.kt

```diff
@@ -121,7 +121,7 @@ class TiltaksgjennomforingValidatorTest : FunSpec({
             .shouldContainExactlyInAnyOrder(ValidationError("oppstart", "Tiltaket må ha løpende oppstartstype"))
     }
 
-    test("skal godta endringer selv om avtale er avbrutt") {
+    test("kan ikke opprette på ikke Aktiv avtale") {
```

Comment:
Jeg tror vi hadde motsatt oppførsel før, men at vi implementerte `skal godta endringer selv om avtale er avbrutt` fordi vi har overtatt data fra Arena der avtalen er avbrutt, men fortsatt har aktive gjennomføringer. Synes vi burde ta en sjekk i dataen vår for å se om det fortsatt er tilfellet

Reply:
Dropp det, så testen lengre nede :+1: